charts/celeborn/templates/worker-statefulset.yaml

```diff
@@ -71,6 +71,26 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+      {{- $dirs := .Values.volumes.worker }}
+      {{- if eq "hostPath" (index $dirs 0).type }}
+      - name: chown-celeborn-worker-volume
+        image: alpine:3.18
+        imagePullPolicy: {{ .Values.image.pullPolicy | default "IfNotPresent" }}
+        securityContext:
+          runAsUser: 0
+        command:
+        - chown
+        - {{ .Values.securityContext.runAsUser | default 10006 }}:{{ .Values.securityContext.runAsGroup | default 10006 }}
+        {{- range $dir := $dirs }}
+        - {{ $dir.path }}
+        {{- end}}
+        volumeMounts:
```

Comment:
There is duplicate volumnMounts for celeborn-worker-vol-{{ index }} in main containers, can we remove that? Otherwise we may have consistency problem.

Reply:
I'm a little confused.

Please correct me if you have any questions, Kubernetes volumes for pod and each container in pod should 
declare volumeMounts.

> To use a volume, specify the volumes to provide for the Pod in .spec.volumes and declare where to mount those volumes into containers in .spec.containers[*].volumeMounts.

[Volumes](https://kubernetes.io/docs/concepts/storage/volumes/)

So `duplicate volumnMounts` are not repetitive, but necessary, equivalent to referring to a variable.

```yaml
containers:
    image: registry-vpc.cn-hangzhou.aliyuncs.com/emr-ack/spark-mingji:v0.2.1-incubating
    imagePullPolicy: Always
    name: celeborn
    volumeMounts:
    - mountPath: /opt/celeborn/conf
      name: celeborn-volume
      readOnly: true
    - mountPath: /mnt/disk1
      name: celeborn-worker-vol-0
    - mountPath: /mnt/disk2
      name: celeborn-worker-vol-1
    - mountPath: /mnt/disk3
      name: celeborn-worker-vol-2
    - mountPath: /mnt/disk4
      name: celeborn-worker-vol-3
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: kube-api-access-fpmg7
      readOnly: true
  initContainers:
    image: alpine:3.18.0
    imagePullPolicy: Always
    name: chown-celeborn-worker-volume
    volumeMounts:
    - mountPath: /mnt/disk1
      name: celeborn-worker-vol-0
    - mountPath: /mnt/disk2
      name: celeborn-worker-vol-1
    - mountPath: /mnt/disk3
      name: celeborn-worker-vol-2
    - mountPath: /mnt/disk4
      name: celeborn-worker-vol-3
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: kube-api-access-fpmg7
      readOnly: true
```

Reply:
OK, thanks for explanation.